### PR TITLE
Add PHP Send Message example

### PR DESCRIPTION
**Pull request by thinkverse**
This PR adds a PHP version of the `sendMessage()` example as described in (https://github.com/appwrite/appwrite/issues/3955).

The PR uses the same channels and services as the other send messages PRs, that way the examples will be consistent with each other. Services used are `Mailgun` and `Twilio`.

Hope the PR is satisfactory, looking forward to the review. 🙂 Screenshots are included below. 👍

I could not get autoloading inside the function to work, therefore I had to use `include_once` instead. I tried `PSR-4` and `classmap`, but the runtime refused to find the files in `src`, even though they were copied to the `.tar` successfully by Docker. Might be something to look further into at a later date.

<details><summary><strong>Screenshots of working function.</strong>
</summary>

### SMS

*Image resized*

phone-resized

### Email

Screen Shot 2022-10-07 at 9 09 34 PM

### Twitter

Screen Shot 2022-10-07 at 8 56 39 PM

### Discord

Screen Shot 2022-10-07 at 8 43 38 PM

</details>

Closes https://github.com/appwrite/appwrite/issues/3955

### CODE REVIEW

File: php/send_message/index.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    try {
+        $payload = \json_decode($req['payload'], true);
+    } catch (\Exception $e) {
+        $res->json([
+            'status' => false,
+            'message' => $e->getMessage(),
+        ]);
+    };
+
+    $notificator = match (strtolower($payload['type'])) {
```
Comment (stnguyen90) [inline php/send_message/index.php:21]: There's a warning if an array key that doesn't exist is accessed:

```
Warning: Undefined array key \"type\" in /usr/code-start/index.php on line 21
```

Would you please improve this to not have warnings?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
This is beautiful code! Great PR! 🤯 We left some comments during the review, please check them out.

File: php/send_message/index.php (hunk 1 above)
Comment (thinkverse) [reply php/send_message/index.php:28]: Resolved in https://github.com/open-runtimes/examples/pull/62/commits/8c25307002c7889295d21bb72d0fe2e194450fa1 👍

**Review by stnguyen90 — APPROVED** [review]
Awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**